index.bs

```diff
@@ -4990,6 +4990,85 @@ Instead, in step three, the comparison on the host is relaxed to accept hosts on
+            1. Pass |facetId| and |appId| to the FIDO algorithm for [=determining if a caller's
+                FacetID is authorized for an AppID=]. If the latter algorithm rejects |appId| then
+                return a "{{SecurityError}}" {{DOMException}} and terminate the
+                [creating a new credential](#sctn-createCredential) algorithm as well as these steps.
+
+                Note: In practice, several implementations do not implement steps four and onward of the algorithm for [=determining if a caller's FacetID is authorized for an AppID=]. Instead, in step three, the comparison on the host is relaxed to accept hosts on the [=same site=].
+
+            1. Otherwise, continue with normal processing.
+
+        1. Just prior to [invoking authenticatorMakeCredential](#CreateCred-InvokeAuthnrMakeCred) perform these steps:
+            1. If |authenticator| supports the U2F protocol [[!FIDO-U2F-Message-Formats]], then [=list/for each=]
+                [credential descriptor](#dictdef-publickeycredentialdescriptor) |C| in |excludeCredentialDescriptorList|:
+                1. Check whether |C| was created using U2F on |authenticator| by sending a
+                    `U2F_AUTHENTICATE` message to |authenticator|
+                    whose "five parts" are set to the following values:
+
+                        :   |control byte|
+                        ::  `0x07` ("check-only")
+
+                        :   <var ignore>challenge parameter</var>
+                        ::  32 random bytes
+
+                        :   <var ignore>application parameter</var>
+                        ::  SHA-256 hash of |appId|
+
+                        :   <var ignore>key handle length</var>
+                        ::  The length of <code>|C|.{{PublicKeyCredentialDescriptor/id}}</code> (in bytes)
+
+                        :   <var ignore>key handle</var>
+                        ::  The value of <code>|C|.{{PublicKeyCredentialDescriptor/id}}</code>, i.e., the [=credential id=].
+
+                1. If |authenticator| responds with `message:error:test-of-user-presence-required` (i.e., success):
+                    cease normal processing of this |authenticator| and indicate in a platform-specific manner
+                    that the authenticator is inapplicable. For example, this could be in the form of UI, or
+                    could involve requesting [=user consent=] from |authenticator| and, upon receipt, treating
+                    it as if the authenticator had returned {{InvalidStateError}}. Requesting [=user consent=]
+                    can be accomplished by sending another `U2F_AUTHENTICATE` message to |authenticator| as
+                    above except for setting |control byte| to `0x03` ("enforce-user-presence-and-sign"),
+                    and ignoring the response.
+
+            1. Continue with normal processing.
+
+: Client extension output
+:: None.
```

Comment:
@jcjones [Raises a good point](https://github.com/w3c/webauthn/pull/1244#issuecomment-508227696) - this should probably be a constant `true` output, like the `appid` extension had previously (and like `authnSel` and `biometricPerfBounds` have currently).

Reply:
@agl Makes a good [counter-argument](https://github.com/w3c/webauthn/pull/1244#issuecomment-509316913) to this.